src/std/rec/compressRecord.dbd.pod

```diff
@@ -7,43 +7,387 @@
 # in file LICENSE that is included with this distribution. 
 #*************************************************************************
 
-=title Compress Record (compress)
+=title Compression Record (compress)
 
-...
+The data compression record is used to collect and compress data from arrays.
+When the INP field references a data array field, it immediately compresses the
+entire array into an element of an array using one of several algorithms,
+overwriting the previous element. If the INP field obtains its value from a
+scalar-value field, the compression record will collect a new sample each time
+the record is processed and add it to the compressed data array as a circular
+buffer.
 
-=head2 Record-specific Menus
-
-=head3 Menu compressALG
-
-The ALG field which uses this menu controls the compression algorithm used.
-
-...
-
-=menu compressALG
-
-...
-
-=head2 Parameter Fields
-
-The record-specific fields are described below.
+The INP link can also specify a constant; however, if this is the case, the
+compression algorithms are ignored, and the record support routines merely
+return after checking the FLNK field.
 
 =recordtype compress
 
-...
-
 =cut
 
 menu(compressALG) {
 	choice(compressALG_N_to_1_Low_Value,"N to 1 Low Value")
 	choice(compressALG_N_to_1_High_Value,"N to 1 High Value")
 	choice(compressALG_N_to_1_Average,"N to 1 Average")
+	choice(compressALG_N_to_1_Median,"N to 1 Median")
 	choice(compressALG_Average,"Average")
 	choice(compressALG_Circular_Buffer,"Circular Buffer")
-	choice(compressALG_N_to_1_Median,"N to 1 Median")
```

Comment:
Did you move the `N to 1 Median` choice entry? Doing that changes the ENUM value of that choice, which could break some existing databases that might write a number to the ALG field so I would prefer to leave it where it was (compatibility has to win over giving in to OCD tendencies!).

Reply:
Ops! sorry I didn't think that the order matters here. Will revert it in next commit.